### DISCUSSION TITLE
Packaging Google Sheet drivers into the latest superset docker image - is there a premade image?

### DISCUSSION THREAD

**Discussion by AmarNirgunkar** [apache/superset#24891] [Q&A / Help]
Hello there,

Can anyone share the latest superset production docker image with Google Sheets drivers preinstalled and configured?

**Comment by sfirke** [comment]
Hello, as this is not a bug report it would be better suited to the project's GitHub Discussions or the Superset Slack.  I have converted it to a Discussion, if someone knows of such an image they can post it here.

The project does not package versions of the application with database drivers, that's left up to end users.  The latest production release is version 2.1.0.  You could add Google Sheets drivers to it by either:

- Building your own image starting from apache/superset:2.1.0 with example code to install database drivers from [this blog post](https://preset.io/blog/2020-06-01-connect-superset-google-sheets/) or [this other one](https://preset.io/blog/2020-05-18-install-db-drivers/)
- Adding it to a requirements-local.txt file in the `/docker/` directory and then running `docker-compose up`, the logs should show the package getting installed

The db driver package for Google Sheets is listed here: https://superset.apache.org/docs/databases/installing-database-drivers

There is a lot of Google Sheets info scattered around the docs but it doesn't seem to have a single authoritative source; if you have any suggestions for improving the docs once you get this working, please let us know!
